### PR TITLE
fix(mep): Handle empty primary query

### PR DESCRIPTION
**Pull request by wmak**
- This fixes a bug that caused tuple() in tuple(tuple())
   - This was cause the primary query would get no results back, but we were still adding something to the groupby_values since groupby_key would be `()`, resulting in groupby_values = `[()]`
   - This meant that on L1849 where we check if there are groupby_values we'd get a truthy value when there wasn't actually any values to groupby
- Fixes SNUBA-2SS

### CODE REVIEW

File: tests/sentry/search/events/test_builder.py [hunk 1]
```diff
@@ -1707,6 +1708,24 @@ def test_no_error_if_aggregates_disallowed_but_no_aggregates_included(self):
...
+        result = MetricsQueryBuilder(
+            self.params,
+            selected_columns=[
+                "p50()",
+                "count_unique(user)",
+            ],
+            allow_metric_aggregates=False,
+            use_aggregate_conditions=True,
+        ).run_query("test")
+        print(result["data"])
```
Comment (dashed) [inline tests/sentry/search/events/test_builder.py:1722]: Stray print

Comment (wmak) [inline tests/sentry/search/events/test_builder.py:1722]:
```suggestion
```

Comment (dashed) [reply tests/sentry/search/events/test_builder.py]: that was lazy :)

**Review by dashed — APPROVED** [review]

File: tests/sentry/search/events/test_builder.py (hunk 1 above)
Comment (wmak) [reply tests/sentry/search/events/test_builder.py]: ;) could've saved me time by suggesting from the start

**Pull request merged**